apps/cms/src/@types/Product.ts

```diff
@@ -0,0 +1,24 @@
+export class Product {
+  constructor(
+    public id: string = "",
+    public name: string = "",
+    public price: number = 0, // Price of the product in cents ($19 = 1900).
+    public images: string[] = [], // URL of the images.
+    public sizes: ProductSizeTypes[] = [],
+    public productCategory: ProductCategoryType = "",
+    public isAvailable?: boolean
+  ) {}
+}
```

Comment:
Is it necessary to make this a `class` (will there be methods)?
Would it make more sense to declare it as a type instead?

Reply:
Hey Wen Xu, for this class i think Ivan has created it mainly just to create some mock data for the products front end page. The proper products types and api will be implemented by the backend team in the coming week. So for the time being is ok to commit with this mock version?

Reply:
> Is it necessary to make this a `class` (will there be methods)? Would it make more sense to declare it as a type instead?

Hey Wen Xu, for this class i think Ivan has created it mainly just to create some mock data for the products front end page. The proper products types and api will be implemented by the backend team in the coming week. So for the time being is ok to commit with this mock version?

Reply:
@chanbakjsd It must be a class for keys in payload columns.